content/applications/services/helpdesk.rst

```diff
@@ -90,7 +88,6 @@ manually on each ticket individually, or through :guilabel:`Automatic Assignment
 :guilabel:`Automatic Assignment` checkbox to enable this feature for the team.
 
 .. image:: helpdesk/helpdesk-visibility-assignment.png
-   :align: center
    :alt: View of a Helpdesk team settings page emphasizing the automatic assignment features in Odoo
          Helpdesk.
 
```

Comment:
For lines 99 & 102, I would capitalize the words after the colon, since they're full sentences.